src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -43,6 +43,8 @@ public class TestProject
 
         public Dictionary<string, string> AdditionalProperties { get; } = new Dictionary<string, string>();
 
+        public Dictionary<string, string> AdditionalItems { get; } = new Dictionary<string, string>();
```

Comment:
This name feels quite general for something that only supports a single item of a given type with no metadata.

Reply:
Let me know if you have a better name :). I was following the pattern PropertyGroup -> AdditionalProperties, and ItemGroup -> AdditionalItems :)

Reply:
Since this is just tests, I'll let it go. The name is good, but the type is not general enough to express all items. Like you can't add two excludes with this.